classes/Settings.js

```diff
@@ -13,42 +17,40 @@ class Settings {
 
   async get() {
     try {
-      // retrieve _config.yml and social-media.yml
-    	const configResp = new Config(this.accessToken, this.siteName)
+      // retrieve _config.yml, index.md, and footer.yml
```

Comment:
Do we retrieve index.md as well?

Reply:
removed in [380b956](https://github.com/isomerpages/isomercms-backend/pull/60/commits/380b956d14a5c5c3c15ccf3e2091ec21cf396769)